daisy_workflows/image_build/debian/build.py

```diff
@@ -17,25 +17,13 @@
 
 Parameters (retrieved from instance metadata):
 
-bootstrap-vz-manifest: The version of bootstrap-vz to retrieve and use.
```

Comment:
Its personal preference- but to keep consistency with daisy variables, it makes it easier (and less error prone) to keep them consistent.